tests/test_clock.py

```diff
@@ -30,3 +32,57 @@ def test_negative_numbers_raise_ValueError(self):
     def test_now_is_timezone_aware(self):
         """now() should always return a tz-aware datetime."""
         self.assertIsNotNone(self.clock.now().tzinfo)
+
+
+class TimerTest(unittest.TestCase):
+
+    def setUp(self):
+        self.mock_clock = mock.Mock()
+
+    def test_timer_duration_exceeded(self):
+        duration = datetime.timedelta(hours=3 * 24)
+        self.mock_clock.now.side_effect = [
+            datetime.datetime(
+                2016, 7, 23, 10, 51, 9, 928000, tzinfo=pytz.utc),
+            datetime.datetime(
+                2016, 7, 27, 10, 51, 9, 928000, tzinfo=pytz.utc)
+        ]
+        self.assertTrue(clock.Timer(self.mock_clock, duration).expired())
+
+    def test_timer_duration_matched(self):
+        duration = datetime.timedelta(hours=3 * 24)
+        self.mock_clock.now.side_effect = [
+            datetime.datetime(
+                2016, 7, 23, 10, 51, 9, 928000, tzinfo=pytz.utc),
+            datetime.datetime(
+                2016, 7, 26, 10, 51, 9, 928000, tzinfo=pytz.utc)
+        ]
+        self.assertFalse(clock.Timer(self.mock_clock, duration).expired())
```

Comment:
I expected this  to be `True`. If we set a 5 minute timer at 1pm, wouldn't we want it to expire at 1:05pm rather than 1:05:00.000001pm?